### ISSUE TITLE
Could not connect a Database client.

### ISSUE THREAD

**Issue by dfolgado** [dunglas/symfony-docker#312]
Everytime I wanted  to connect the PHPStorm Database client, I was not able, I was getting this error:

`Postgresql : Connection refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections`

**Comment by ernestwisniewski** [comment]
you should expose ports, ex:

`###> doctrine/doctrine-bundle ###
  database:
    image: postgres:${POSTGRES_VERSION:-13}-alpine
    environment:
      POSTGRES_DB: ${POSTGRES_DB:-app}
      # You should definitely change the password in production
      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-ChangeMe}
      POSTGRES_USER: ${POSTGRES_USER:-symfony}
    volumes:
      - db-data:/var/lib/postgresql/data:rw
      # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!
      # - ./docker/db/data:/var/lib/postgresql/data:rw
    ports:
      - "5433:5432" <-----------
###< doctrine/doctrine-bundle ###`

**Comment by ernestwisniewski** [comment]
you should expose ports in docker-compose.yaml:

`
###> doctrine/doctrine-bundle ###
...
    ports:
      - "5433:5432"
`

**Comment by dfolgado** [comment]
Yes, thank you.  Can we add this permanently to the `docker-compose.yml` ?

**Comment by maxhelias** [comment]
No, this service is added dynamically by Flex as needed and has been designed to be compatible with the Symfony CLI and this project.
